.github/workflows/ci.yml

```diff
@@ -10,6 +10,14 @@ jobs:
       - uses: actions/setup-python@v4
         with:
           python-version: "3.10"
-      - run: python -m pip install -r requirements.txt
-      - run: dbt deps
-      - run: dbt build
+          cache: 'pip'
```

Comment:
Not necessary for this PR, but this gives a nice little speed boost if dependencies haven't changed since last run.